lib/Elastica/ResultSet.php

```diff
@@ -253,8 +249,6 @@ public function current()
     public function next()
     {
         ++$this->_position;
-
-        return $this->current();
```

Comment:
next() is not meant to return something.

Reply:
also it's not documented. and it would also not be really useful because it might return false or null or whatever when the iterator reaches the end which is what happens as it happens before valid is called.